packages/patternlab/styleguide/source/_patterns/03-organisms/page-banner/page-banner.twig

```diff
@@ -1,121 +1,45 @@
-            {% if pageBanner.titleSubText %}
-              <abbr title="{{ pageBanner.title }}"> {{ pageBanner.titleSubText }}</abbr>
-            {% endif %}
-          </h1>
-        </div>
-        {% if pageBanner.primaryLink %}
-          {% set decorativeLink = pageBanner.primaryLink %}
-          {% if pageBanner.primaryLink.href %}
-            {% include "@atoms/decorative-link.twig" %}
-          {% else %}
-            <p class="ma__page-banner__unlinked-title">
-              {{pageBanner.primaryLink.text}}
-            </p>
-          {% endif %}
-          {% if pageBanner.primaryLink.description %}
-            <div class="ma__page-banner__description">
-              {{pageBanner.primaryLink.description}}
-            </div>
-          {% endif %}
-        {% endif %}
-        {% if pageBanner.secondaryLink %}
-          {% set decorativeLink = pageBanner.secondaryLink %}
-          {% if pageBanner.secondaryLink.href %}
-            {% include "@atoms/decorative-link.twig" %}
-          {% else %}
-            <p class="ma__page-banner__unlinked-title">
-              {{pageBanner.secondaryLink.text}}
-            </p>
-          {% endif %}
-          {% if pageBanner.secondaryLink.description %}
-            <div class="ma__page-banner__description">
-              {{pageBanner.secondaryLink.description}}
-            </div>
-          {% endif %}
-        {% endif %}
-        {% if pageBanner.description %}
-          <p class="ma__page-banner__description">
-            {{ pageBanner.description }}
-            {% if pageBanner.descriptionSubText %}
-              <span class="ma__page-banner__descriptionSubText">
-                {{ pageBanner.descriptionSubText }}
-              </span>
-            {% endif %}
-          </p>
-        {% endif %}
-        {% if pageBanner.buttons %}
-          <div class="ma__page-banner__buttons-wrapper">
-            {% if pageBanner.buttonsLabel %}
-              <span  class="ma__page-banner__buttons-label">{{pageBanner.buttonsLabel}}</span>
-            {% endif %}
-            <ul class="ma__page-banner__buttons">
-              {% for button in pageBanner.buttons %}
-                <li class="ma__page-banner__button">
-                  {% include "@atoms/01-buttons/button.twig" %}
-                </li>
-              {% endfor %}
-            </ul>
-          <div>
-        {% endif %}
-      </div>
-    </div>
-  </div>
-</section>
+{% set theme = pageBanner.theme ? " ma__page-banner--" ~ pageBanner.theme : " ma__page-banner--c-primary" %}
+{# If there's no image, don't let the subTitle and description text span the full width of the page #}
+{% set fullWidthText = pageBanner.image %}
+
+<div class="ma__page-banner{{theme}}">
+  <div class="ma__page-banner__container">
+    <div class="ma__page-banner__content">
+      {% set pageHeader = pageBanner.pageHeader  %}
+      {# {% set pageHeader = pageHeader|merge({"fullWidth": fullWidthText}) %} #} 
```

Comment:
Help wanted:

I'm trying to pass the boolean value to pageHeader.fullWidth whenever the PageHeader component is used within PageBanner, but It's breaking the build with this error:
`PHP Fatal error:  Uncaught Twig\\Error\\RuntimeError: The merge filter only works with arrays or "Traversable", got "NULL" as first argument in...`

Reply:
To replicate, uncomment this line of code

Reply:
I think this means that pageHeader is null or empty, found this snippet online, 
{% set pageHeader = pageHeader|default({})|merge({"fullWidth": fullWidthText}) %}
this is setting pageheader to be emtyp if it's not set, you can try this

Reply:
Thank you!! This fixes the issue!